Jenkinsfile

```diff
@@ -12,9 +12,11 @@
 def runTests = true
 
 // Only keep the 10 most recent builds.
-properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator',
-                                                                        numToKeepStr: '50',
-                                                                        artifactNumToKeepStr: '20']]])
+// NOTE: With Pipeline 1.14 and later combined with Jenkins 1.642 and later, this may need to be changed to
```

Comment:
No, I think you are still confused.

Leave as `jenkins.model.BuildDiscarderProperty`. When some future Pipeline release merging https://github.com/jenkinsci/workflow-plugin/pull/321 is made (which will _not_ be 1.14), and you are running 1.642.x or later (which that release will require), then you can drop the FQN. In the meantime, keep it.